docs/quick-start/configure-mongodb.txt

```diff
@@ -22,7 +22,7 @@ Configure Your MongoDB Connection
 
       .. code-block:: php
 
-         'default' => env('DB_CONNECTION', 'mongodb'),
+         'default' => 'mongodb',
```

Comment:
If people want to update their default connection, it might be in the env var `DB_CONNECTION` in the `.env` file.

Reply:
This line should set the default connection to MongoDB even if the `DB_CONNECTION` variable is set to something else in `.env`. But I updated the admonition (in query-builder.txt) to give two options on how to set MongoDB as the connection, if that helps clarify

Reply:
I remain convinced that this is the not the right way to change the default connection.

Reply:
I remain convinced that this is the not the right way to change the default connection. @alcaeus any opinion?

Reply:
Laravel's default configuration uses env variables, and I agree that we shouldn't deviate too far from this.

Reply:
Okay, I changed these code snippets back to using env variables. Since we don't explain the .env file or its variables in the quick start, I made a [ticket](https://jira.mongodb.org/browse/DOCSP-38890) to add this info.